releases/0.2.1/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 0.2.1 Release
+layout: releases
+custom_css: releases
+---
+
+# {{ page.title }}
+
```

Comment:
@plbt5 , were you intending to list the first 0.2.1-specific proposal on this page?

Reply:
Change Proposal 38 was applied in the history of the CASE `master` branch.  Can you add that to the 0.2.1 and 0.4.0 pages?  We apparently missed listing it as a change in 0.4.0.